src/app/tests/suites/certification/Test_TC_S_2_6.yaml

```diff
@@ -357,7 +356,7 @@ tests:
           set to 20000 (20s) and no extension field sets."
       PICS: S.S.C00.Rsp
       verification: |
-          scenesmanagement add-scene 0x0000 0x01 20000 scene1 [] 1 1
+          ./chip-tool scenesmanagement add-scene 0x0000 0x01 20000 scene1 [] 1 1
```

Comment:
In TC-S-2.6 - The `./chip-tool ` prefix should be removed from the command from test step 3a onwards. Same for all the test step after 3a.  Please refer the VS document.

Reply:
Removed for every step after interactive start.